AutomatedTesting/Gem/PythonTests/AWS/conftest.py

```diff
@@ -81,3 +82,26 @@ def teardown():
     request.addfinalizer(teardown)
 
     return pytest.cdk_obj
+
+
+
+@pytest.fixture(scope='function')
+def aws_credentials(request: pytest.fixture, aws_utils: pytest.fixture, profile_name: str):
```

Comment:
seems like this change should be applied to all automation test?

Reply:
This change is applied to all the automation tests for AWS gems. It's defined in the config file under the AWS root folder.